ra2ce/runners/direct_analysis_runner.py

```diff
@@ -115,7 +71,7 @@ def run(
             )
             _results.append(_result_wrapper)
 
-            self._save_result(_result_wrapper)
+            AnalysisResultWrapperExporter().export_result(_result_wrapper)
```

Comment:
What is not taken into account is that the various analysis modules/packages export much more data than is reflected in the `ResultWrapper`. I think this needs some rethinking.